app/test/logic/Contacts/VCard/VCard.ts

```diff
@@ -58,22 +62,50 @@ export default {
         person.streetAddresses.add(makeContactEntry(entry));
       }
     }
-    if (vcard.entries.note) {
-      person.notes = vcard.entries.note[0].value;
-    }
-    if (vcard.entries.org) {
-      person.company = vcard.entries.org[0].values[0];
-      person.department = vcard.entries.org[0].values[1];
-    }
+    person.notes = vcard.entries.note?.[0].value || "";
```

Comment:
I think you mean `??` ?

Reply:
Sure, that works. (Values are `string | undefined`, so the result is the same either way.)

Reply:
```suggestion
    person.notes = vcard.entries.note?.[0].value ?? "";
```